core/plugins/org.polarsys.capella.core.platform.sirius.sirius.validation/src/org/polarsys/capella/core/platform/sirius/sirius/validation/ddiagram/I_23_AbstractReplaceElementResolver.java

```diff
@@ -0,0 +1,94 @@
+package org.polarsys.capella.core.platform.sirius.sirius.validation.ddiagram;
```

Comment:
Copyright

Reply:
Added on I_23_Abstract.... & DescriptionParserHelper that was also missing copyright